xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -0,0 +1,694 @@
+/*
+ *      Copyright (C) 2016 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "ContextMenuItem.h"
+#include "cores/AudioEngine/Engines/ActiveAE/AudioDSPAddons/ActiveAEDSP.h"
+#include "epg/EpgInfoTag.h"
+#include "pvr/addons/PVRClients.h"
+#include "pvr/channels/PVRChannel.h"
+#include "pvr/PVRGUIActions.h"
+#include "pvr/PVRManager.h"
+#include "pvr/recordings/PVRRecording.h"
+#include "pvr/recordings/PVRRecordingsPath.h"
+#include "settings/Settings.h"
+#include "pvr/timers/PVRTimers.h"
+#include "utils/URIUtils.h"
+
+#include "PVRContextMenus.h"
```

Comment:
Do we have something on this topic in our guidelines? At work we always follow the Google Coding Guidelines, see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes. There the order is
1. header file corresponding to this cpp file
2. C system files
3. C++ system files
4. Other libraries' header files
5. Own header files